blocks/library/button/style.scss

```diff
@@ -37,3 +37,9 @@ $blocks-button__line-height: $big-font-size + 6px;
 		text-align: right;
 	}
 }
+
+.wp-block-button .wp-block-button__link {
+    white-space: normal;
+    word-break: break-all;
+    height: auto; 
```

Comment:
Is `height: auto` needed?

Reply:
I think we can remove it because it does not affects the output.